docs/accumulo-export-queue.md

```diff
@@ -19,8 +19,8 @@ limitations under the License.
 ## Background
 
 The [Export Queue Recipe][1] provides a generic foundation for building export mechanism to any
-external data store. The [AccumuloExporter] provides an implementation of this recipe for
-Accumulo. The [AccumuloExporter] is located the `fluo-recipes-accumulo` module and provides the
+external data store. The [AccumuloConsumer] provides an export consumer for writing to
```

Comment:
A bit of background.  The name I like most is AccumuloExporter. I like this name because it implies sending things out from Fluo.  However this name is currently taken by a class that's deprecated in this PR.

The super type for AccumuloConsumer is ExportConsumer (which is a new type in this PR).